common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java

```diff
@@ -20,7 +20,14 @@
      */
     @JsonProperty("default")
     @Deprecated
-    DEFAULT, // 2
+    DEFAULT(false), // 2
     @JsonProperty("camel")
-    CAMEL // 3
+    CAMEL(true); // 3
+
+    final boolean requiresSubType;
```

Comment:
I'm not sure - right now we don't need to know this information, but if we do, it can be added. The only other place using this would be `Endpoint` and it can already access this value.